<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1583940962442' stop='1583941022463'>
    <name>Login test of webappsecurity page</name>
    <title>Login test of webappsecurity page</title>
    <test-cases>
        <test-case start='1583940962445' status='broken' stop='1583941022451'>
            <name>redirect URL</name>
            <title>redirect URL</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-80.0.3987.132'/>
            </parameters>
            <steps/>
            <attachments>
                <attachment title='Execution video' source='3215ef3d-5c4a-4d53-9dd5-056e004882c5-attachment.mp4' type='video/mp4' size='146'/>
            </attachments>
            <failure>
                <message>Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/soumyasen/Automation/WebdriverIO_Automation/tests/login.test.js)</message>
                <stack-trace>Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/soumyasen/Automation/WebdriverIO_Automation/tests/login.test.js)
                    at Test.Runnable._timeoutError (/Users/soumyasen/Automation/WebdriverIO_Automation/node_modules/mocha/lib/runnable.js:463:10)
                    at Timeout.&lt;anonymous> (/Users/soumyasen/Automation/WebdriverIO_Automation/node_modules/mocha/lib/runnable.js:267:24)
                    at listOnTimeout (internal/timers.js:549:17)
                    at processTimers (internal/timers.js:492:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>